LibreNMS/Authentication/MysqlAuthorizer.php

```diff
@@ -20,8 +20,9 @@ public function authenticate($credentials)
         $username = $credentials['username'] ?? null;
         $password = $credentials['password'] ?? null;
 
-        $hash = User::thisAuth()->where(['username' => $username])->value('password');
-        $enabled = User::thisAuth()->where(['username' => $username])->value('enabled');
+        $user_data = User::thisAuth()->where(['username' => $username])->value('password', 'enabled');
```

Comment:
you need select, but if you want to support enabled in all authorizers, you'll need to check enabled in LegacyUserProvider::validateCredentials().  I'm not sure if that will invalidate cookie auth though.

Reply:
fixed query :-)

i think "enabled" should be available for local Authentication (mysql based) only, because LDAP Controller does this itself.
But not sure how to check in "resources/views/user/form.blade.php" and "resources/views/user/index.blade.php" if used Authenticationmethod is 'mysql' or 'http_auth' (which means local Authentication)

So i would insert in *.blade.php an Authenticationtypefilter for (not) showing "enabled"

Reply:
@murrant 
query is fixed ...